neo/SmartContract/InteropService.cs

```diff
@@ -537,9 +538,10 @@ private static bool Contract_Call(ApplicationEngine engine)
 
             StackItem method = engine.CurrentContext.EvaluationStack.Pop();
             StackItem args = engine.CurrentContext.EvaluationStack.Pop();
+            string methodStr = method.GetString();
             ContractManifest currentManifest = engine.Snapshot.Contracts.TryGet(engine.CurrentScriptHash)?.Manifest;
 
-            if (currentManifest != null && !currentManifest.CanCall(contract.Manifest, method.GetString()))
+            if (currentManifest != null && !currentManifest.CanCall(contract.Manifest, methodStr))
```

Comment:
@neo-project/core @erikzhang current Manifest could be null? i think that it is a MUST

```suggestion
            if (currentManifest == null || !currentManifest.CanCall(contract.Manifest, methodStr))
```